xl/xenops_server_xenlight.cppo.ml

```diff
@@ -553,17 +553,17 @@ module HOST = struct
           with Xenlight.Host.End_of_file ->
             []
         in
-        let raw = String.concat "" (List.rev (read_lines ())) in
+        let raw = String.concat "" (List.rev (read_lines ())) |> Bytes.unsafe_of_string in
```

Comment:
I am uncomfortable with this as we are modifying it below, I'll change it to use `of_string`